content/client-lib-development-guide/features.textile

```diff
@@ -1336,6 +1339,10 @@ h4. ChannelProperties
 * @(CP2)@ The attributes of @ChannelProperties@ consist of:
 ** @(CP2a)@ @attachSerial@ string - contains the last @channelSerial@ received in an @ATTACHED@ @ProtocolMesage@ for the channel, see "RTL15a":#RTL15a
 
+h4. ChannelDetails
+
+* @(CHD1)@ A @ChannelDetails@ is a type that represents information for a channel including channelID, name, status and occupancy 
```

Comment:
We need to add `Channel#status` and the `ChannelDetails` type to the IDL section at the bottom of this document.

Reply:
This has been added